examples/main.tf

```diff
@@ -566,6 +582,7 @@ resource "instaclustr_cluster" "example-cadenceav" {
   cluster_name = "testcluster-cadenceav"
   node_size = "CAD-DEV-t3.small-5"
   data_centre = "US_WEST_2"
+  data_centre_custom_name = "NetAppHQ"
```

Comment:
There's one more place that needs to be updated (missed it before) - https://github.com/instaclustr/terraform-provider-instaclustr/blob/master/docs/resources/cluster.md

Reply:
done